dev/KaairaGupta/data_loader.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import numpy as np
+from sklearn.model_selection import train_test_split
+from sklearn.preprocessing import StandardScaler
+from sklearn.discriminant_analysis import LinearDiscriminantAnalysis as LDA
+
+
+
+def load_split_preprocessed_data(i):
+
+    data = pd.read_csv("../../datasets/winequality.csv")
+    Count_Fraud_transacation = len(data["recommend"]==True) 
+    fraud_indices= np.array((data.recommend==True).index)
+    normal_indices = np.array((data.recommend==False).index)
+
+    undersample_data = undersample(normal_indices,fraud_indices,i,Count_Fraud_transacation,data)
```

Comment:
What is the meaning of the variable name `Count_Fraud_transacation`?
In the context of the wine dataset this variable naming is confusing.

Reply:
Renamed the variables for better intution